### PR TITLE
update from https://www.facebook.com/syntezator/ 15.12.2020

### PR DESCRIPTION
**Pull request by velmyshanovnyi**
update from https://www.facebook.com/syntezator/ 15.12.2020

### CODE REVIEW

File: userdict/src/dictionary.txt [hunk 1]
```diff
@@ -394,20 +420,25 @@ wba/i
...
 облад+ун*
 област+{ей,ям,ями,ях}
 облік+ові
 обсє/i
 оддал+ік
 од{ин,на,не,ні,но} +одн*
 од{ин,на,не,ні,но} {без,біля,в,від,для,до,з,за,із,коло,на,наперед,перед,поперед,про,у,через} +одн*
+оп+іка {і,й,та} піклування
+оп+іки {і,й,та} піклування
+оп+іку {і,й,та} піклування
```
Comment (velmyshanovnyi) [inline userdict/src/dictionary.txt:433]: дублювання
431
432
433

Comment (alex19EP) [reply userdict/src/dictionary.txt:433]: can you fix it?

Comment (alex19EP) [reply userdict/src/dictionary.txt:433]: on second glance. this a different words. I'm not sure what the problem is.

File: userdict/src/dictionary.txt [hunk 2]
```diff
@@ -1,9 +1,11 @@
-; This dictionary for Ukrainian voices which are part of RHVoice Speech Synthesizer (https://github.com/Olga-Yakovleva/RHVoice) contains the rules for the correct pronunciation of Ukrainian words and abbreviations.
-; This dictionary is distributed under LGPL v2.1 or later.
-; Copyright (C) 2017-2020 Synthesizer of Ukrainian Speech team (https://facebook.com/syntezator)
-; Coordinator and author:
-; Volodymyr Pyrih <vp88.mobile@gmail.com>
-
+; 01 ; This dictionary for Ukrainian voices which are part of RHVoice Speech Synthesizer (https://github.com/RHVoice) contains the rules for the correct pronunciation of Ukrainian words and abbreviations.
```
Comment (alex19EP) [inline userdict/src/dictionary.txt:1]:
```suggestion
; 01 ; This dictionary for Ukrainian voices which are part of RHVoice Speech Synthesizer (https://github.com/RHVoice/RHVoice) contains the rules for the correct pronunciation of Ukrainian words and abbreviations.
```

I also don’t know why 01, 02, etc. are needed.

**Review by alex19EP — CHANGES_REQUESTED** [review]
since I don't know Ukrainian, I can only hope that everything is fine here.

File: userdict/src/dictionary.txt (hunk 2 above)
Comment (velmyshanovnyi) [reply userdict/src/dictionary.txt:1]: Цифри  для того, чтоб сортировка по алфавиту работала, и ети строки оставались в нужному порядке сверху

Comment (velmyshanovnyi) [reply userdict/src/dictionary.txt:1]: Пример: Документ правится например в блокноте, в конец документа добавляются новие правила. Потом сортируются по алфавиту, потом удаляются копии строк, потом комит на гитхаб

Comment (alex19EP) [reply userdict/src/dictionary.txt:1]: now i understand. thanks.
when you address my suggestion i will merge this pr.

Comment (VovaMobile) [reply userdict/src/dictionary.txt:1]: Hi developers,
I am official author of the Ukrainian pronunciation dictionary for Ukrainian voices of RHVoice speech synthesizer.
Today i commited latest official dictionary in new pull request.
First version of this dictionary included in the latest stable version of this synthesizer.
I continued developing, updating and improving this dictionary.
Official dictionary has only alphabet structure without digits and oder.

**Pull request closed without merging**